Develop an SQL schema for SOPs

```diff
@@ -1,29 +1,4351 @@
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab"."Peptide Section_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2967 (class 0 OID 0)
+-- Dependencies: 223
+-- Name: Peptide Section_id_seq; Type: SEQUENCE OWNED BY; Schema: MzTab; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab"."Peptide Section_id_seq" OWNED BY "MzTab"."Peptide Section".id;
+
+
+--
+-- TOC entry 209 (class 1259 OID 17263)
+-- Name: Protein section; Type: TABLE; Schema: MzTab; Owner: postgres
+--
+
+CREATE TABLE "MzTab"."Protein section" (
+    accession text NOT NULL,
+    description text NOT NULL,
+    taxid integer NOT NULL,
+    species text NOT NULL,
+    database text NOT NULL,
+    database_version text NOT NULL,
+    search_engine text[] NOT NULL,
+    best_search_engine_score_number text NOT NULL,
+    best_search_engine_score text NOT NULL,
+    search_engine_score_number text NOT NULL,
+    ms_run_number text NOT NULL,
+    search_engine_score_ms_run text NOT NULL,
+    reliability integer NOT NULL,
+    num_psms_ms_run_number text NOT NULL,
+    num_psms_ms_run integer NOT NULL,
+    num_peptides_distinct_ms_run_number text NOT NULL,
+    num_peptides_distinct_ms_run integer NOT NULL,
+    num_peptides_unique_ms_run_number text NOT NULL,
+    num_peptides_unique_ms_run integer NOT NULL,
+    ambiguity_members text[] NOT NULL,
+    modifications text NOT NULL,
+    uri text NOT NULL,
+    go_terms text[] NOT NULL,
+    protein_coverage double precision NOT NULL,
+    protein_abundance_assay_number text NOT NULL,
+    protein_abundance_assay double precision NOT NULL,
+    protein_abundance_study_variable_number text NOT NULL,
+    protein_abundance_study_variable double precision NOT NULL,
+    protein_abundance_stdev_study_variable_number text NOT NULL,
+    protein_abundance_stdev_study_variable double precision NOT NULL,
+    protein_abundance_std_error_study_variable_number text NOT NULL,
+    protein_abundance_std_error_study_variable double precision NOT NULL,
+    "mzTab-ID_REFS" text NOT NULL,
+    id integer NOT NULL
+);
+
+
+ALTER TABLE "MzTab"."Protein section" OWNER TO postgres;
+
+--
+-- TOC entry 222 (class 1259 OID 18171)
+-- Name: Protein section_id_seq; Type: SEQUENCE; Schema: MzTab; Owner: postgres
+--
+
+CREATE SEQUENCE "MzTab"."Protein section_id_seq"
+    AS integer
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
+
+
+ALTER TABLE "MzTab"."Protein section_id_seq" OWNER TO postgres;
+
+--
+-- TOC entry 2968 (class 0 OID 0)
+-- Dependencies: 222
+-- Name: Protein section_id_seq; Type: SEQUENCE OWNED BY; Schema: MzTab; Owner: postgres
+--
+
+ALTER SEQUENCE "MzTab"."Protein section_id_seq" OWNED BY "MzTab"."Protein section".id;
+
+
+--
+-- TOC entry 206 (class 1259 OID 17205)
+-- Name: Metadata_metabolomics; Type: TABLE; Schema: MzTab-M; Owner: postgres
+--
+
+CREATE TABLE "MzTab-M"."Metadata_metabolomics" (
```

Comment:
Are all of these columns type `text`?  It seems that there should be some integers or floats/doubles.

Reply:
done